### DISCUSSION TITLE
LaserGRBL problem with board

### DISCUSSION THREAD

**Discussion by PhoenixWood** [arkypita/LaserGRBL#1630] [Q&A]
Hello, I’m trying to do some laser engraving on a wood project and when I get to a certain point it says there’s an issue with my board. Would that me an issue in my laser engraver? I have an Ortur laser. Thank you

**Comment by gmmanonymus111** [comment]
What is the error message you receive? Depending on that, it could be caused by a faulty board, cable, USB-to-Serial adapter etc
